src/components/tabs/tabs.stories.ts

```diff
@@ -174,3 +174,6 @@ export const TabChilrenWithPercentageHeights = (): string => html`
     </calcite-tab>
   </calcite-tabs>
 `;
+TabChilrenWithPercentageHeights.parameters = {
+  chromatic: { delay: 1000 }
```

Comment:
Sidebar: I think we could extract this delay in a helper module for reuse.

Reply:
Yeah, I have it sprinkled around on other stories too. The only thing is I'd rather not delay all stories, only the ones we need to

Reply:
I meant extracting the delay value, so we would import the same reference.

Reply:
Gotcha. I've been putting different delay values at this point to see what works without eating too much time. But once I figure that out I'll create a const